fluent/src/scope.js

```diff
@@ -0,0 +1,29 @@
+export default class Scope {
+  constructor(
+    bundle,
+    errors,
+    args,
+    parameterized = false,
+    dirty = new WeakSet()
+  ) {
+    this.bundle = bundle;
```

Comment:
It would be nice in the future to replace `Scope.bundle` with all the private members of `FluentBundle` which are needed by the resolver. I.e. add `Scope.messages`, `Scope.terms` etc. This would reduce the API surface of the bundle. I'm leaving this as a follow-up, though, to keep this PR on topic.